dev-util/colm/colm-0.13.0.5.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="COmputer Language Manipulation"
+DESCRIPTION="Colm is a programming language designed for the analysis and \
+transformation of computer languages. Colm is influenced primarily by TXL. It \
+is in the family of program transformation languages."
+HOMEPAGE="https://www.colm.net/open-source/colm/"
+COPYRIGHT="2006-2016 Adrian Thurston"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.colm.net/files/colm/colm-$portVersion.tar.gz"
+CHECKSUM_SHA256="33e624677176958eaad76ebe6c391a68a0b4728fec8cc039efa1316f525f408c"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```

Comment:
@extrowerk, could you have a look at the sources and see if colm can be built with gcc2 ?
If it cannot, then you should mark x86_gcc2 as broken:
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

BTW, I just did some metrics, and it turns out that there are many more recipes using

```
commandSuffix=$secondaryArchSuffix
commandBinDir=$binDir
if [ "$targetArchitecture" = x86_gcc2 ]; then
	commandSuffix=
	commandBinDir=$prefix/bin
fi
```
than recipes using
```
commandBinDir=$binDir
commandSuffix=$secondaryArchSuffix
if [ "$targetArchitecture" = x86_gcc2 ]; then
	commandBinDir=$prefix/bin
	commandSuffix=
fi
```
So if you swap the order of `commandSuffix` and `commandBinDir` (to have `commandSuffix` before `commandBinDir`) it would be cool :)
Thanks!

BTW, if colm can be built with gcc2 then you should drop the bloc with `commandSuffix` and `commandBinDir`.